### PR TITLE
Fix/extract literals

### PR DESCRIPTION
**Pull request by l4l**
### Description of the Change

Extract some literals to the constant variables. Note that it doesn't cover irohad/model directory at all, affected files are below

iroha-cli:
- impl/grpc_response_handler.cpp
- impl/query_response_handler.cpp
- interactive/impl/interactive_query_cli.cpp

irohad/ametsuchi:
- /impl/postgres_wsv_query.cpp
- /impl/storage_impl.cpp

irohad/consensus:
- yac/impl/yac.cpp

irohad/main:
- application.cpp
- iroha_conf_loader.hpp

irohad/network:
- impl/block_loader_impl.cpp

irohad/torii:
- command_client.cpp

irohad/validation:
- impl/stateless_validator_impl.cpp

libs:
- crypto/keys_manager_impl.cpp
- crypto/keys_manager_impl.hpp
- logger/logger.cpp

### Benefits

Easier fixing, cleaner code

### Possible Drawbacks 

None

### CODE REVIEW

File: irohad/network/impl/block_loader_impl.cpp [hunk 1]
```diff
@@ -111,15 +117,15 @@ nonstd::optional<Block> BlockLoaderImpl::retrieveBlock(
...
+    log_->error(kPeerRetrieveFail);
     return nonstd::nullopt;
   }
 
-  auto it = std::find_if(
-      peers.value().begin(), peers.value().end(),
-      [pubkey](auto peer) { return peer.pubkey == pubkey; });
+  auto it = std::find_if(peers.value().begin(),
+                         peers.value().end(),
+                         [pubkey](auto peer) { return peer.pubkey == pubkey; });
```
Comment (Warchant) [inline irohad/network/impl/block_loader_impl.cpp]: const auto& peer
&pubkey

File: irohad/validation/impl/stateless_validator_impl.cpp [hunk 2]
```diff
@@ -32,11 +32,16 @@ namespace iroha {
       log_ = logger::log("SLV");
     }
 
+    const char *kCryptoVerificationFail = "crypto verification broken";
+    const char *kFutureTimestamp =
+        "Timestamp broken: send from future {}. Now {}";
```
Comment (Warchant) [inline irohad/validation/impl/stateless_validator_impl.cpp]: is broken
sent

**Comment by Warchant** [comment]
Codecov says that coverage is bad, this is because none of failure cases in postgres are covered. It is not the part of this PR.

File: irohad/validation/impl/stateless_validator_impl.cpp [hunk 3]
```diff
@@ -32,11 +32,16 @@ namespace iroha {
       log_ = logger::log("SLV");
     }
 
+    const char *kCryptoVerificationFail = "crypto verification broken";
```
Comment (Warchant) [inline irohad/validation/impl/stateless_validator_impl.cpp]: and maybe, "invalid signature"?

Comment (l4l) [reply irohad/validation/impl/stateless_validator_impl.cpp]: Yeah, agree

**Comment by Warchant** [comment]
Also look at this:
>BLOCKER iroha_conf_loader.hpp#L47: Using namespace are not allowed in header files. rule

File: iroha-cli/interactive/impl/interactive_query_cli.cpp [hunk 4]
```diff
@@ -108,11 +108,15 @@ namespace iroha_cli {
       create_result_menu();
     }
 
+    void printMenu(MenuPoints menu) {
```
Comment (Warchant) [inline iroha-cli/interactive/impl/interactive_query_cli.cpp]: pass by const ref

**Pull request merged**